README.md

```diff
@@ -35,6 +40,9 @@ Find another group to work with. The other group should be students who neither
 Return to your original locations.  In this step all groups will act like Designers.  Another vital aspect of design thinking is open-minded brainstorming about potential solutions that will fulfill the Customer's needs.   Review your notes about the Customer's needs.  Use cards from the Things, Services, Human Actions, and Feedback decks to *design two (2) or more different solutions* to the Customer's needs.
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
+Both our solutions use elements of virtual reality. Here are our two solutions:
+1. google glasses that record the views, sound. the recorded data can be transffered to a phone memory. Everything can be exported to vr set with gloves. vr set is a helmet itself making the sound as close.
+2. Smart camera paired with a smart room. The camera records sights, smells, sounds, temperature, and humidity 360. Then These senses would be projected in the smart room in your home.
```

Comment:
That would have to be a high-tech camera if it were to record more than sounds and video!